### PR TITLE
feat: unlock password reset

### PR DESCRIPTION
**Pull request by bumi**
### Describe the changes you have made in this PR

This adds a more descriptive error message and allows the user to reset their accounts to start over

### Link this PR to an issue

#1169 #401 

### Type of change (Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes (If any)

image

### How has this been tested?

Try to unlock with an invalid password

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -22,12 +22,12 @@
         }
       },
       "set_password": {
-        "title": "Protect your wallet",
-        "description": "Your wallet is securely encrypted with a password and needs to be unlocked before usage.",
+        "title": "Set your an unlock password",
```
Comment (lisabaut) [inline src/i18n/locales/en/translation.json:25]: Typo => "Set an unlock password"

File: src/i18n/locales/en/translation.json [hunk 2]
```diff
@@ -22,12 +22,12 @@
...
+        "description": "Your Lightning account data is securely encrypted with an unlock password. Do not forget this password you need it to unlock the Alby Extension",
```
Comment (lisabaut) [inline src/i18n/locales/en/translation.json:26]: Better with a new sentence in between:

"Your Lightning account data is securely encrypted with an unlock password. Do not forget this password! You need it to unlock the Alby Extension (in this browser)"

File: src/extension/background-script/actions/setup/reset.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
-import state, { DEFAULT_SETTINGS } from "../../state";
+import state from "../../state";
 
 const reset = async (message, sender) => {
   state.setState({
-    settings: DEFAULT_SETTINGS,
```
Comment (lisabaut) [inline src/extension/background-script/actions/setup/reset.js:5]: 👍

File: src/i18n/locales/en/translation.json [hunk 4]
```diff
@@ -188,6 +188,10 @@
       "help_contact": {
         "part1": "Need help? Contact",
         "part2": "Alby Support"
+      },
+      "unlock_error": {
+        "help": "Your account data is encrypted with your unlock password. If you really forgot your unlock password you have to reset and add your Lightning account again.",
```
Comment (lisabaut) [inline src/i18n/locales/en/translation.json:193]: Small change:

"Your account data is encrypted with your unlock password. If you really forgot your unlock password you have to reset **your data** and add your Lightning account again."

**Review by lisabaut — APPROVED** [review]
Lightning speed ⚡ And no extra Modal 🙂 
See some suggestions below.

File: src/i18n/locales/en/translation.json (hunk 4 above)
Comment (bumi) [reply src/i18n/locales/en/translation.json:193]: wasn't sure if we then need to describe what data? 
and if we say "account data" what does that mean?

Comment (lisabaut) [reply src/i18n/locales/en/translation.json:193]: Yes, true. Might sound scary.

**Pull request merged**